File: webwork-web/web-core/src/types/Role.ts
/** 角色数据类型 */
export declare class Role {
    /** ID */
    id?: number;
    /** 名称 */
    name?: string;
    /** 编码 */
    code?: string;
    /** 状态 */
    status?: number;
    /** 排序 */
    sort?: number;
    /** 备注 */
    remark?: string;
    /** 父级ID */
    parentId?: number;
    /** 创建时间 */
    createTime?: string;
    /** 其它属性 */
    [key: string]: any;
}

File: webwork-user/src/main/java/com/github/wephotos/webwork/user/entity/Role.java
package com.github.wephotos.webwork.user.entity;

import java.util.Date;

import com.baomidou.mybatisplus.annotation.TableName;
import lombok.*;

/**
 * 角色
 *
 * @author chengzi
 * @date 2021-01-25 16:34
 */
@Getter
@Setter
@ToString
@AllArgsConstructor
@NoArgsConstructor
@TableName(value = "sys_role")
public class Role {
    /**
     * 主键
     */
    private Integer id;
    /**
     * 名称
     */
    private String name;
    /**
     * 编码
     */
    private String code;
    /**
     * 状态
     */
    private Integer status;
    /**
     * 上级(单位或应用)
     */
    private Integer parentId;
    /**
     * 排序
     */
    private Integer sort;
    /**
     *备注
     */
    private String remark;
    /**
     * 创建时间
     */
    private Date createTime;
}
